magento/pwa-studio
# PWA-2128: Finishing tasks for docs migration to adobeio

## Description

This PR completes the remaining content and source code template formatting needed before redirecting users to the new adobeio docs site. 

## Related Issue

NA. Internal story.

## Acceptance

### Verification Stakeholders

- @jcalcaben

## Verification Steps

Build the jekyll site from the branch to validate changes and ensure that no errors occur.
